test/test_playbooks/fixtures/host-3.yml

```diff
@@ -0,0 +1,318 @@
+      - nosniff
+      x-download-options:
+      - noopen
+      x-frame-options:
+      - sameorigin
+      x-permitted-cross-domain-policies:
+      - none
+      x-powered-by:
+      - Phusion Passenger 4.0.18
+      x-request-id:
+      - 93d25ba6-eb42-45c6-9ef2-208e7387e24e
+      x-runtime:
+      - '0.158095'
+      x-xss-protection:
+      - 1; mode=block
+    status:
+      code: 200
+      message: OK
+- request:
+    body: !!python/unicode '{"power_action": "state"}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '25'
+      User-Agent:
+      - python-requests/2.21.0
+      content-type:
+      - application/json
+    method: PUT
+    uri: https://or.deploy1.dev.atix/api/v2/hosts/43/power
+  response:
+    body:
+      string: !!python/unicode '{"power":"off"}'
+    headers:
+      apipie-checksum:
+      - 8ad89255b081b6d6ea905a7af33ca1767e4c467f
+      cache-control:
+      - max-age=0, private, must-revalidate
+      connection:
+      - Keep-Alive
+      content-length:
+      - '15'
+      content-security-policy:
+      - 'default-src ''self''; child-src ''self''; connect-src ''self'' ws: wss:;
+        img-src ''self'' data: *.gravatar.com; script-src ''unsafe-eval'' ''unsafe-inline''
+        ''self''; style-src ''unsafe-inline'' ''self'''
+      content-type:
+      - application/json; charset=utf-8
+      date:
+      - Tue, 19 Mar 2019 12:18:37 GMT
+      etag:
+      - W/"f89e4c8af2d57ebb35dfb5643fbabd3f"
+      foreman_api_version:
+      - '2'
+      foreman_version:
+      - 1.20.1
+      keep-alive:
+      - timeout=5, max=10000
+      server:
+      - Apache/2.4.6 (CentOS)
+      set-cookie:
+      - request_method=PUT; path=/; secure; HttpOnly; SameSite=Lax
+      - _session_id=46f8d6067cfbb2c7db0591e7ec7987a4; path=/; secure; HttpOnly; SameSite=Lax
+      status:
+      - 200 OK
+      strict-transport-security:
+      - max-age=631139040; includeSubdomains
+      x-content-type-options:
+      - nosniff
+      x-download-options:
+      - noopen
+      x-frame-options:
+      - sameorigin
+      x-permitted-cross-domain-policies:
+      - none
+      x-powered-by:
+      - Phusion Passenger 4.0.18
+      x-request-id:
+      - a3515f09-fe02-4d5c-8af5-1fcf64c2eabd
+      x-runtime:
+      - '0.313847'
+      x-xss-protection:
+      - 1; mode=block
+    status:
+      code: 200
+      message: OK
+- request:
+    body: !!python/unicode '{"power_action": "on"}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '22'
+      User-Agent:
+      - python-requests/2.21.0
+      content-type:
+      - application/json
+    method: PUT
+    uri: https://or.deploy1.dev.atix/api/v2/hosts/43/power
+  response:
+    body:
+      string: !!python/unicode '{"power":{"task_state":"success"}}'
```

Comment:
I think this is not the state, we want to return from the ansible module.

Reply:
"This is not the state we are looking for"

Reply:
fixed